libsquish/Makefile

```diff
@@ -1,31 +1,470 @@
+# CMAKE generated file: DO NOT EDIT!
+# Generated by "Unix Makefiles" Generator, CMake Version 3.12
```

Comment:
This file has been replaced by a generated version and must be reverted to its original state.  Another good reason to only do out-of-source builds :)